library/core/src/num/f16.rs

```diff
@@ -509,12 +509,15 @@ impl f16 {
     /// #![feature(f16)]
     /// # // FIXME(f16_f128): remove when `extendhfsf2` and `truncsfhf2` are available
     /// # #[cfg(target_os = "linux")] {
+    /// # // FIXME(f16_f128): remove when `__gnu_h2f_ieee` and `__gnu_f2h_ieee` are available
+    /// # #[cfg(not(all(target_arch = "mips", target_os = "linux")))] {
```

Comment:
I think you can probably simplify these to just `not(target_arch = "mips")`, unless there are known non-linux mips targets that do have the symbols.

Reply:
Actually even simpler, could you just make the existing gates `all(target_arch = "x86_64", target_arch = "linux")`? So we don't run into this again on other less common platforms. We are testing these functions more thoroughly in the unit tests so I'm not too concerned about doctests running on all platforms.

Reply:
Non-linux MIPS behavior is unknown to me in this case, so I wanted to be conservative, but I think your proposal makes sense. Done!